### ISSUE TITLE
"Create per track CUE sheet" doesn't work properly

### ISSUE THREAD

**Issue by Jojojoe1** [flacon/flacon#196]
When I open cue file as below,
--------------------------------------------------------------------------------------------------------------
```
REM GENRE Sample
REM DATE 2000
REM DISCID 12345
REM COMMENT "ExactAudioCopy v0.99pb5"
PERFORMER "Sample Performer"
TITLE "Sample Title"
FILE "Sample1.wav" WAVE
  TRACK 01 AUDIO
    TITLE "Title1"
    PERFORMER "Sample Performer"
    INDEX 01 00:00:00
FILE "Sample2.wav" WAVE
  TRACK 02 AUDIO
    TITLE "Title2"
    PERFORMER "Sample Performer"
    INDEX 01 00:00:00
FILE "Sample3.wav" WAVE
  TRACK 03 AUDIO
    TITLE "Title3"
    PERFORMER "Sample Performer"
    INDEX 01 00:00:00
```
--------------------------------------------------------------------------------------------------------------

Then convert it to flac with "Create per track CUE sheet", output cue file is below
(TRACK 02 location is wrong, it should be under FILE "02 - Title2.flac" )
--------------------------------------------------------------------------------------------------------------
```
REM GENRE "Sample"
REM DATE 2000
REM DISCID 12345
PERFORMER "Sample Performer"
TITLE "Sample Title"
FILE "01 - Title1.flac" WAVE
  TRACK 01 AUDIO
    TITLE "Title1"
    INDEX 00 00:00:00
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Title2"
FILE "02 - Title2.flac" WAVE
    INDEX 01 00:00:00
  TRACK 03 AUDIO
    TITLE "Title3"
FILE "03 - Title3.flac" WAVE
    INDEX 01 00:00:00
```

**Comment by Jojojoe1** [comment]
[Sample.zip](https://github.com/flacon/flacon/files/10905964/Sample.zip)

**Comment by SokoloffA** [comment]
Included in release 11.0.0
